mmv1/third_party/terraform/services/compute/resource_compute_ha_vpn_gateway_test.go

```diff
@@ -0,0 +1,74 @@
+package compute_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccComputeHaVpnGateway_updateLabels(t *testing.T) {
```

Comment:
Test failed due to error:
```
resource_compute_ha_vpn_gateway_test.go:20: Step 3/4 error: Check failed: Check 2/2 error: google_compute_ha_vpn_gateway.ha_gateway1: Attribute 'labels.testupdated' expected "testupdated", got ""
```

From the [debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-12866/artifacts/1020dae7-8c4e-4e43-a0d8-31ca8077a05f/recording/TestAccComputeHaVpnGateway_updateLabels.log), it doesn't seem the labels get updated.